### ISSUE TITLE
allow move frames in XXXsheet

### ISSUE THREAD

**Issue by Q210108** [opentoonz/opentoonz#4338]
just a little suggestion for a best productivity

**Comment by Q210108** [comment]
> Keyframes can move - just click on the keyframe icon so it turns blue. Then drag. To move drawings + keyframes, hold ctrl when you make the selection. Then release ctrl and click/drag on the dragbar.

sorry, i meant frames

**Comment by RodneyBaker** [comment]
Welcome to Opentoonz!

Levels (images exposures) can currently be moved in the xsheet and timeline... that's rather the whole point of the xsheet.
You may want to join an Opentoonz discord where you can interact live with other Opentoonz users and work out the basic processes.

Basically we move exposures in xsheet/timeline in a number of ways; drag/dropping or using keystrokes (Q and W) to move level frames forward and back.  Many other methods exist as well depending on the mode and tool being used at the moment.
 
I must assume you are primarily interested in the 'frames' of levels as the greater Frames those are placed in themselves never change... that is a linear progress from 0 to some number I currently forget... not infinity but more frames than generally would be reasonable.

In short, your feature request has been implemented in a number of different ways.  
I'll tag as 'needs info' and perhaps you can refine the request.

**Comment by RodneyBaker** [comment]
There have been users that have desired the older subxsheet UI elements from Toonz to be brought back.
REF:  #4338

**Comment by RodneyBaker** [comment]
Closing as lacking information to determine path forward to resolve the issue as frames currently can be moved in the xsheet/timeline.  @Q210108 If appropriate, please open a new request with additional information or feel free to continue the discussion here.
